pages/common/gh.md

```diff
@@ -24,9 +24,9 @@
 
 `gh pr create`
 
-- View a pull request in the browser:
+- Create a pull request to a specified branch instead of the default:
 
-`gh pr view --web {{pr_number}}`
+`gh pr create --base {{dev}}`
```

Comment:
I think this is an specific example, and we have the pages `gh-pr` and `gh-pr-create` for more specific arguments.

Reply:
Apologies, did not look :(, let me update